test/functional/ui/screen.lua

```diff
@@ -79,6 +79,7 @@ end
 --- @field win_position table<integer,table<string,integer>>
 --- @field float_pos table<integer,table>
 --- @field cmdline table<integer,table>
+--- @field cmdline_abort? boolean
```

Comment:
Didn't think it necessary to add this as an ext_keys, to avoid having to add this to a bunch of existing tests.

Reply:
Not sure I like this actually, considering doing this anyways; adding the `abort` state as a field in `self.cmdline[level]` instead.